AIMSICD/src/main/res/layout/layout_request_gps.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
using `layout_` as prefix for the filename is redundant, better use `dialog_` as it maches with the existing `activity_` and `fragment_` prefixes